torch/_dynamo/eval_frame.py

```diff
@@ -197,6 +197,11 @@ def __setstate__(self, state):
         self.__dict__ = state
         self._initialize()
 
+    def __getattribute__(self, name):
+        if name == "training":
+            return getattr(self._orig_mod, name)
+        return super().__getattribute__(name)
+
```

Comment:
Perhaps:
```suggestion
    @property
    def training(self):
        return getattr(self._orig_mod, name)
```

The prior method will add overhead on every single attribute access.

Reply:
Needs a little more work to get the setter working.